src/test/java/net/william278/profanitycheckerapi/ProfanityCheckerTests.java

```diff
@@ -10,6 +10,13 @@ public class ProfanityCheckerTests {
     @Test
     public void givenSentenceContainingProfanity_testIsProfane() {
         Assertions.assertTrue(checker.isTextProfane("This is a fucking test sentence"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("_54848498niggakedsamda"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("Sh1tface"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("Twat"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("Cunt"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("n1gga"));
+        Assertions.assertTrue(checker.isTextProfaneBypass("nnnnnnnnnnnnnnnnnnnnnnn1gggggggggggggggggaaaaaaaaaaaaaa56446465495"));
+        System.out.println(checker.getTextProfanityLikelihoodBypass("nnnnnnnnnnnnnnnnnnnnnnn1gggggggggggggggggaaaaaaaaaaaaaa56446465495"));
```

Comment:
Could we change these test cases away from using a racial slur? Of course, testing a profanity filter inherently means we have to pass profane words, I'm just a little uneasy using it for our testing data in this case (the result is the same in terms of profanity, racial slur or not)